dist-assets/linux/mullvad-gui-launcher.sh

```diff
@@ -8,5 +8,15 @@ else
     SANDBOX_FLAG=""
 fi
 
+SUPPORTED_COMPOSITORS="sway river"
```

Comment:
I think you need spaces at the start and end of this string. Otherwise `" sway "` will never exist in this string.

Reply:
This is done in the `echo " $SUPPORTED_COMPOSITORS "` line below.

Reply:
Ah, of course. My own little test did not do that dance. Bash is such a hack :facepalm: This really should be an array. But bash is bash, so OK.